### ISSUE TITLE
--file-version does not change `fixed.file_version`

### ISSUE THREAD

**Issue by powellnorma** [tc-hib/go-winres#19]
This causes Window to display `0.0.0.0` under "Details > File Version". For some reason `--product-version` "works", even though it doesn't set `fixed.product_version` either (checked via resource hacker)

**Comment by gbatanov** [comment]
Yes, I would like --file-version and --product-version to adjust the corresponding fields in winres.json
